consent/api/v1/views.py

```diff
@@ -233,6 +249,8 @@ def delete(self, request):
         self.set_consent_state(False, user_id, course_id, enterprise_customer_uuid)
         provided = consent_provided(user_id, course_id, enterprise_customer_uuid)
         required = consent_required(user_id, course_id, enterprise_customer_uuid)
+        if not enterprise_customer.catalog_contains_course_run(request.user, course_id):
+            required = False
```

Comment:
Same here I believe with the else block suggestion? Hopefully I didn't fundamentally misunderstand something.